unstructured/ingest/processor.py

```diff
@@ -107,10 +107,11 @@ def process_documents(
     verbose: bool,
     dest_doc_connector: t.Optional[BaseDestinationConnector] = None,
 ) -> None:
+    languages = partition_config.ocr_languages.split("+") if partition_config.ocr_languages else []
```

Comment:
how about we define the joiner character/pattern somewhere in the repo and document it? 
also since the definition for `ocr_languages` is str, not optional str we can safely just write 
```
languages = partition_config.ocr_languages.split("+")
```

Reply:
The joiner character right now is documented in the CLI help string: [unstructured/ingest/cli/interfaces.py#L101](https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/ingest/cli/interfaces.py#L101):
```python
            click.Option(
                ["--ocr-languages"],
                default="eng",
                help="A list of language packs to specify which languages to use for OCR, "
                "separated by '+' e.g. 'eng+deu' to use the English and German language packs. "
                "The appropriate Tesseract "
                "language pack needs to be installed."
                "Default: eng",
            ),
```